pkg/build/gobuild.go

```diff
@@ -1022,6 +1083,45 @@ func (g *gobuild) buildOne(ctx context.Context, refStr string, base v1.Image, pl
 		},
 	})
 
+	delvePath := "" // path for delve in image
+	if g.debug {
+		// get delve locally
+		delveBinary, err := getDelve(ctx, *platform)
+		if err != nil {
+			return nil, fmt.Errorf("building Delve: %w", err)
+		}
+		defer os.RemoveAll(filepath.Dir(delveBinary))
+
+		delvePath = "/usr/bin/" + filepath.Base(delveBinary)
```

Comment:
It might make sense to put it in `/ko-app/dlv` just so it's clear it's owned by `ko` and not by the base image, in case it's already available there.

Reply:
That makes sense to me.

Reply:
Pushed an update!